Formula/fakeroot.rb

```diff
@@ -51,3 +54,17 @@ def install
     assert_match version.to_s, shell_output("#{bin}/fakeroot -v")
   end
 end
+
+__END__
+diff --git a/wrapawk_macosx b/wrapawk_macosx
+index f783219..2301a06 100644
+--- a/wrapawk_macosx
++++ b/wrapawk_macosx
+@@ -10,6 +10,7 @@ BEGIN{
+   print WARNING > headerfile;
+   print "#ifndef WRAPPED_H" > headerfile;
+   print "#define WRAPPED_H" > headerfile;
++  print "#include <spawn.h>" > headerfile;
+   print "#define MY_GLUE2(a,b) a ## b"      > headerfile;
+   print "#define MY_DEF(a) MY_GLUE2(my_,a)" > headerfile;
+   print WARNING > deffile;
```

Comment:
🙋 I have an activated Salsa account.

By the way, would this be more appropriate:

```diff
diff --git a/wrapfunc.inp b/wrapfunc.inp
index f592aad..fa2ff40 100644
--- a/wrapfunc.inp
+++ b/wrapfunc.inp
@@ -48,10 +48,11 @@ fgetattrlist;int;(int fd, void *attrList, void *attrBuf, size_t attrBufSize, uns
 #if MAC_OS_X_VERSION_MIN_REQUIRED >= MAC_OS_X_VERSION_10_5
 getattrlist$UNIX2003;int;(const char *path, void *attrList, void *attrBuf, size_t attrBufSize, unsigned long options);(path, attrList, attrBuf, attrBufSize, options)
 #endif
 #endif
 #if MAC_OS_X_VERSION_MIN_REQUIRED >= MAC_OS_X_VERSION_10_5
+#include <spawn.h>
 lstat$INODE64;int;(const char *file_name, struct stat *buf);(file_name, buf)
 stat$INODE64;int;(const char *file_name, struct stat *buf);(file_name, buf)
 fstat$INODE64;int;(int fd, struct stat *buf);(fd, buf)
 posix_spawn;int;(pid_t * __restrict pid, const char * __restrict path, const posix_spawn_file_actions_t *file_actions, const posix_spawnattr_t * __restrict attrp, char *const argv[ __restrict], char *const envp[ __restrict]);(pid, path, file_actions, attrp, argv, envp)
 posix_spawnp;int;(pid_t * __restrict pid, const char * __restrict path, const posix_spawn_file_actions_t *file_actions, const posix_spawnattr_t * __restrict attrp, char *const argv[ __restrict], char *const envp[ __restrict]);(pid, path, file_actions, attrp, argv, envp)
```

Tested locally; seems to work.

Reply:
Yea, that looks better.